apps/css-workshop/pages/select.html

```diff
@@ -503,6 +503,383 @@ <h2>Default</h2>
 
     <br />
 
+    <h2>Borderless</h2>
+    <section id="demo-borderless">
+
+      <label
+        class="iui-input-grid"
+        id="demo-borderless-singular"
+      >
+        <div class="iui-input-label iui-required">Select label</div>
```

Comment:
If LabeledSelect supports it, we should have a story for it too.

Also, we should think about the best way to label this and document it (can be done in the future if we're unsure). We are not even showing a label in https://github.com/iTwin/appui/pull/733

![image](https://github.com/iTwin/iTwinUI/assets/9084735/2e6e40db-bd7e-4a5f-818d-3bf92d1f49a9)

Reply:
> If LabeledSelect supports it, we should have a story for it too.

The reasoning was that although we support it, we don't recommend users to overuse it. So, we thought to not include a story for this in `LabeledSelect` but just in `Select`.

Reply:
In that case we should maybe exclude this prop from `LabeledSelect`, similar to other restrictions (mentioned in https://github.com/iTwin/iTwinUI/pull/1886#discussion_r1511207171).

Reply:
Let's exclude the prop from `LabeledSelect`.

Reply:
`styleType` is excluded from `LabeledSelect` in types and in code ([one of the commits](https://github.com/iTwin/iTwinUI/pull/1886/commits/d9895da725c582554c9e3e8183b536081ce807bf)).